### PR TITLE
[PAN-2968] Add explicit instructions to execute the call on majority o…

### PR DESCRIPTION
**Pull request by bgravenorst**
## PR description
Add explicit instructions to execute the call on majority of signers/validators.

## Fixed Issue(s)
PAN-2968

### CODE REVIEW

File: docs/Consensus-Protocols/Clique.md [hunk 1]
```diff
@@ -93,9 +93,15 @@ The JSON-RPC methods to add or remove signers are:
 * [clique_getSigners](../Reference/Pantheon-API-Methods.md#clique_getsigners)
 * [clique_discard](../Reference/Pantheon-API-Methods.md#clique_discard)
 
+!!! important
+    Adding or removing a signer is subject to agreement by existing signers. Meaning the command to add or remove a signer must
```
Comment (MadelineMurray) [inline docs/Consensus-Protocols/Clique.md]: I'd reword the first sentence to be active: 
A majority of existing signers must agree on adding or removing a signer. 

And be more specific about what _the command_ is: 
That is, `clique_proposeValidatorVote` must be executed specifying the signer to add or remove on the majority ...

File: docs/Consensus-Protocols/Clique.md [hunk 2]
```diff
@@ -93,9 +93,15 @@ The JSON-RPC methods to add or remove signers are:
...
+    be executed on the majority of signers (greater than 50%) to take effect.
+
 Use [clique_getSignerMetrics](../Reference/Pantheon-API-Methods.md#clique_getsignermetrics) to view signer metrics for a specified block range.
 
-To propose adding a signer, call `clique_propose` specifying the address of the proposed signer and `true`.
+### Adding a Signer
```
Comment (MadelineMurray) [inline docs/Consensus-Protocols/Clique.md:104]: It's much better with these new headings :-)

File: docs/Consensus-Protocols/IBFT.md [hunk 3]
```diff
@@ -132,9 +132,15 @@ The JSON-RPC methods to add or remove validators are:
 * [ibft_proposeValidatorVote](../Reference/Pantheon-API-Methods.md#ibft_proposeValidatorVote)
 * [ibft_discardValidatorVote](../Reference/Pantheon-API-Methods.md#ibft_discardValidatorVote)
 
-Use [ibft_getSignerMetrics](../Reference/Pantheon-API-Methods.md#ibft_getsignermetrics) to view signer metrics for a specified block range.
+!!! important
+    Adding or removing a validator is subject to agreement by existing validators. Meaning the command to add or remove a validator must
```
Comment (MadelineMurray) [inline docs/Consensus-Protocols/IBFT.md]: Same comments re wording as for the Clique change.

**Review by MadelineMurray — CHANGES_REQUESTED** [review]
Edits and I think we need to be clear IBFT 2.0 requiring a super-majority

File: docs/Consensus-Protocols/IBFT.md [hunk 4]
```diff
@@ -132,9 +132,15 @@ The JSON-RPC methods to add or remove validators are:
...
+    be executed on the majority of validators (2/3) to take effect.
```
Comment (ajsutton) [inline docs/Consensus-Protocols/IBFT.md]: Only a majority of validators is required, not 2/3rds.  It's exactly the same code as for Clique.  `number_of_validators / 2 + 1` is required (ie a clear majority).

File: docs/Consensus-Protocols/Clique.md (hunk 1 above)
Comment (bgravenorst) [reply docs/Consensus-Protocols/Clique.md]: Agreed. Fixed it here and for IBFT.Agreed. Fixed it here and for IBFT.

File: docs/Consensus-Protocols/IBFT.md (hunk 4 above)
Comment (NicolasMassart) [reply docs/Consensus-Protocols/IBFT.md]: I discussed that with @ajsutton and it looks like:
- super majority (2/3rd = ~66%) is required to validate a block
- majority (number_of_validators / 2 + 1) is only required to add/remove validators

See highlighted parts in https://media.consensys.net/scaling-consensus-for-enterprise-explaining-the-ibft-algorithm-ba86182ea668

Comment (bgravenorst) [reply docs/Consensus-Protocols/IBFT.md]: Thanks, this clears things up. @ajsutton does Clique also require  the super-majority to validate a block?

Comment (bgravenorst) [reply docs/Consensus-Protocols/IBFT.md]: Feedback from @ajsutton, "No clique proposers just create a block and there are no sign offs."

**Comment by bgravenorst** [comment]
Agreed with Madeline to close this pull request and recreate it to due to conflicts with new navigation structure.
